proto/engine/v1/electra.go

```diff
@@ -1,4 +1,118 @@
 package enginev1
 
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
 type ExecutionPayloadElectra = ExecutionPayloadDeneb
 type ExecutionPayloadHeaderElectra = ExecutionPayloadHeaderDeneb
+
+var (
+	drExample = &DepositRequest{}
+	drSize    = drExample.SizeSSZ()
+	wrExample = &WithdrawalRequest{}
+	wrSize    = wrExample.SizeSSZ()
+	crExample = &ConsolidationRequest{}
+	crSize    = crExample.SizeSSZ()
+)
+
+func (eee *ExecutionBundleElectra) GetDecodedExecutionRequests() (*ExecutionRequests, error) {
+	requests := &ExecutionRequests{}
+
+	if len(eee.ExecutionRequests) != 3 /* types of requests */ {
```

Comment:
This better be a named constant like `LenExecutionRequestsElectra` or similar, this is likely to change per fork.